test/spec/directive.coffee

```diff
@@ -77,6 +77,22 @@ describe 'angular-masonry', ->
     expect(@scope.$watch).not.toHaveBeenCalled()
   )
 
+  it 'should setup a $watch when the reload-on-resize is present', inject(($compile) =>
+    sinon.spy(@scope, '$watch')
+    element = angular.element '<masonry reload-on-resize></masonry>'
+    element = $compile(element)(@scope)
+
+    expect(@scope.$watch).toHaveBeenCalled()
```

Comment:
That looks a bit too generic. I'm worried that this will break if someone else uses `$watch` for a different reason.